.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["es2015", "stage-0"]
-}
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "current"
```

Comment:
At the moment we support node 8, we should change this to 8 so it's not a breaking change.